lib/ansiblelint/utils.py

```diff
@@ -1,3 +1,4 @@
+"""Utilities."""
```

Comment:
Plz put this under the comment.
```suggestion
"""Generic utility helpers."""
```